glide.yaml

```diff
@@ -12,3 +12,6 @@ import:
 - package: github.com/pkg/errors
   version: ~0.7.1
 - package: github.com/spf13/cobra
+  version: 9c28e4bbd74e5c3ed7aacbc552b2cab7cfdfe744
+- package: github.com/spf13/pflag
+  version: 7b17cc4658ef5ca157b986ea5c0b43af7938532b
```

Comment:
It doesn't look like pflag tags releases.  Is there a particular reason you're pinning here instead of at the current tip (spf13/pflag@5ccb023)?  Just because that's the currently-vendored commit?  I'm fine with that, but would like a note to that effect in the commit message.  Something like:

> pflag doesn't tag releases, so I'm pinning this at our currently-vendored commit.  I'm not aware of any reason we can't advance this to a later pflag commit, but haven't done enough research to say for sure.  This commit is only focused on bumping the vendored image-spec.